astropy/io/fits/diff.py

```diff
@@ -249,10 +250,22 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
             are kept in memory or output.  If a negative value is given, then
             numdiffs is treated as unlimited (default: 10).
 
-        tolerance : float, optional
+        rtol : float, optional
             The relative difference to allow when comparing two float values
             either in header values, image arrays, or table columns
-            (default: 0.0).
+            (default: 0.0). Values which satisfy the expression
+
+            .. math::
+
+                \\left| a - b \\right| > \\text{atol} + \\text{rtol} \\cdot \\left| b \\right|
+
+            are considered to be different.
+            The underlying function used for comparison is `numpy.allclose`.
```

Comment:
Could you add a `.. versionchanged:: 2.0`-directive here explaining the change of parameter name `tolerance->rtol`?